docs/meetings/team/2023_02_13.md

```diff
@@ -0,0 +1,75 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 10:05
+
+**End Time** 10:30
+
+**Location** Discord
+
+**Date** February 13, 2023
+
+## 👋 Attendence
+
+1. Veronica
+2. Edouard
+3. Logan
+4. Matt
+
+***Attendence Comments***
+
+- Logan will be not in Kelowna this Friday for medical reasons, but can still do work
+- Ed will be unavailable on Friday, as he's leaving for an international trip Thursday night
+
+## 🃏 Joke of the Day
+
+Did you know crocodiles can grow up to 16 feet? I still don’t believe it though. I’ve only seen ones with 4.
+
+## 📢 Announcements
+
+- Week 20 GitStats due this week on Friday
+- TA will view our demo this Wednesday, February 15th
+- Reading week is next week, so no Gitstats!
+
+## 📅 Last Week (Restrospective)
+
+- We sent the client our pose requests for the artist and asked for each position to be sent individually instead of as
+a batch, but we haven't heard back about it yet
+- Ed got another unit test in last week!
+- We unfortunately can't run a Git Runner on Rhubarb beacuse it's not on the repo. Do we want to ignore these tests?
+Matt & Logan say there should be a solution (e.g., installing a release via command line)
+- Matt got a server image up so our API can run on good hardware instead of locally on our laptops!
+- Logan got the script working for Adobe Illustrator so that managing assets is A LOT easier!
+
+## 📋 This Week
+
+### Assets
+
+- We currently have temporary poses implemented until we receive the real poses from the artist
```

Comment:
@mkuelker to follow up with them today regarding our request - as we still havent heard anything

Reply:
@vnajack we might want to update the request to ask for the new poses in the illustrator file format that we currently have set up - unless we want to manually adjust the layer hierarchy again (which wouldnt be too much trouble)

Reply:
As long as we get it sooner rather than later, I don't think it matters